src/docfx/config/ops/OpsMetadataRuleConverter.cs

```diff
@@ -23,6 +23,7 @@ public static string GenerateJsonSchema(string rulesContent, string allowlistsCo
 
             var schema = new
             {
+                uniquewithindocset = new List<string>(),
```

Comment:
JsonSchema uses camelCase convention. 

Consider `docsetUnique`? This feels more aligned if there are other properties like `globalUnique`.

```suggestion
                docsetUnique = new List<string>(),
```

Reply:
if we already have globalUnique, "docsetUnique" is surely a good idea.